main.js

```diff
@@ -0,0 +1,92 @@
+var inputAmt = document.querySelector('.expense-amount');
+var inputName = document.querySelector('.expense-name');
+var button = document.querySelector('.add-expense');
+var sumSpan = document.querySelector('.expense-total');
+var table = document.querySelector('.expenses');
+var form = document.querySelector('form');
+var sum = [0];
+
+
+// var handleEnter = function(e) {
+//   if (e.keyCode === 13) {
+//     console.log('returnd');
+//     e.preventDefault();
+//   }
+// };
+// inputAmount.addEventListener('keyup', handleEnter);
+// inputName.addEventListener('keyup', handleEnter);
+
+var handleButton = function(e) {
+  // console.log('buttond');
+  e.preventDefault();
+
+  sum.push(Number(inputAmt.value));
```

Comment:
This is a good start to handling data outside of the visual representation of it in the DOM, but like you noticed you run into problems handling removal. One solution might be to grab the amount from the node you're trying to remove, parse that and then subtract it from your sum.